example_vulnerable_code.py

```
@@ -0,0 +1,70 @@
+# Code Quality Issue 1: Broad exception handling
+def risky_operation():
+    try:
+        # Some complex operation
+        result = 1 / 0
+    except:  # Too broad!
+        print("Something went wrong")
+        return None
+
+# Code Quality Issue 2: Missing error handling
+def connect_to_api():
+    response = requests.get("https://api.example.com/data")
```

Ruff (0.11.9)

Undefined name `requests`

(F821)